packages/jss/tests/unit/createGenerateId.js

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
changes here are unrelated, but for some reason after I added tests for styled api, this tests started to fail, something weird started happening with module id, like there is a new moduleId module evaluation, I couldn't fully understand that, but I just started resetinng moduleId here.